cluster/manifests/skipper/deployment.yaml

```diff
@@ -43,7 +43,11 @@ spec:
 {{- end }}
       priorityClassName: "{{ .Cluster.ConfigItems.system_priority_class }}"
       serviceAccountName: skipper-ingress
+{{ if eq .Cluster.ConfigItems.nlb_switch "exec" }}
+      terminationGracePeriodSeconds: {{ 302 }}
```

Comment:
This looks wrong with the `{{` `}}` around the value. I guess it's not caught in e2e because we don't have `nlb_switch "exec"` in e2e.

Reply:
that's interesting how I got to this state :D

Reply:
I guess it's also not wrong to have a text template that returns 302, so why should it fail in e2e tests?

Reply:
> I guess it's also not wrong to have a text template that returns 302, so why should it fail in e2e tests?

You're right, I didn't think about that you just define the int in the template, so it would work fine I guess. Still it's easier to read without `{{` around :)